Gemfile

```diff
@@ -32,6 +32,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'database_cleaner'
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
   gem 'rspec-rails', '>= 2.0.0.beta'
```

Comment:
- [ ] You're expected to have the `rswag` gem installed:

```ruby
gem 'rswag-api'
gem 'rswag-ui'

group :development, :test do
    ...
    gem 'rswag-specs'
end

Reply:
Excuse me @ElijahTobs . This task is an authorization task, the second one of the week. The exercise or rswag is the API documentation task on Friday. 
Thanks for your time.

Reply:
Hello @Metsanpeitto,
Sorry they might have an issue with the dev team they added a different
project guidelines but your projects looks great.